WeakAuras/Prototypes.lua

```diff
@@ -1061,7 +1128,8 @@ WeakAuras.load_prototype = {
       display = L["Player Effective Level"],
       type = "number",
       init = "arg",
-      desc = L["The effective level differs from the level in e.g. Time Walking dungeons."]
+      desc = L["The effective level differs from the level in e.g. Time Walking dungeons."],
+      enable = not WeakAuras.IsClassic() -- WOWCLASSIC TO FIX: hidding it makes an error
```

Comment:
Is that comment still up to date?

Reply:
no, converting WeakAuras.IsClassic to a function seems to fix it, i update this